drobot_motor:
  ros__parameters:
    #way1
    way11: [0.5, 2.5, 0.0] #0
    way12: [0.3, 2.5, 80.0] #1
    way13: [0.3, 2.5, -80.0] #2
    way14: [0.3, 2.5, 160.0] #3

    #way2
    way21: [0.5, 1.55, 0.0] #4
    way22: [0.3, 1.55, 80.0] #5
    way23: [0.3, 1.55, -80.0] #6
    way24: [0.3, 1.55, 160.0] #7

    #way3
    way31: [0.5, 0.45, 0.0] #8
    way32: [0.3, 0.45, 80.0] #9
    way33: [0.3, 0.45, -80.0] #10
    way34: [0.3, 0.45, 160.0] #11

    #way4
    way41: [1.1, 0.4, 0.0] #12
    way42: [1.1, 0.4, 80.0] #13
    way43: [1.2, 0.4, -80.0] #14
    way44: [1.0, 0.4, 160.0] #15

    #way5    
    way51: [1.1, 1.2, 0.0] #16
    way52: [1.1, 1.2, 80.0] #17
    way53: [1.1, 1.2, -80.0] #18
    way54: [1.0, 1.2, 160.0] #19

    #way6
    way61: [1.1, 1.8, 0.0] #20
    way62: [1.1, 1.8, 80.0] #21
    way63: [1.1, 1.8,  -80.0] #22
    way64: [1.0, 1.8, 160.0] #23

    #way7
    way71: [1.1, 2.5, 0.0] #24
    way72: [1.2, 2.5, 80.0] #25
    way73: [1.1, 2.5, -80.0] #26
    way74: [1.0, 2.5, 160.0] #27
    
    #robot
    robot1: [-0.05, 2.6, 0.0] #28
    robot2: [-0.05, 1.6, 0.0] #29
    robot3: [-0.05, 0.6, 0.0] #30

    #store
    store11: [1.5, 2.6, 160.0] #31
    store12: [1.5, 1.7, 160.0] #32
    store21: [1.5, 1.2, 160.0] #33
    store22: [1.5, 0.3, 160.0] #34

    #kiosk
    kiosk11: [0.3, 2.9, 80.0] #35
    kiosk12: [1.0, 2.9, 80.0] #36
    kiosk21: [1.1, 0.0, -80.0] #37
    kiosk22: [0.3, 0.0, -80.0] #38